### PR TITLE
feat: embed storybook in current docs repo

### PR DESCRIPTION
**Pull request by ckifer**
From https://github.com/recharts/recharts/pull/3297 and as an initial step, integrate the storybook with the current docs repo before making more organizational changes in recharts repo

* Add iframe
* Add storybook component and required imports

image

### CODE REVIEW

File: src/locale/zh-CN.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ const map = {
     api: 'API',
     examples: '示例',
     blog: '博客',
+    storybook: 'Storybook (WIP)',
```
Comment (nikolasrieble) [inline src/locale/zh-CN.js:37]: It might be nice to run Work In Progress through google translate and copy paste the chinese output here into the brackets.

File: src/views/IndexView.js [hunk 2]
```diff
@@ -75,8 +75,6 @@ class IndexView extends PureComponent {
           <iframe
             title="star"
             src="https://ghbtns.com/github-btn.html?user=recharts&repo=recharts&type=star&count=true&size=median"
-            frameBorder="0"
-            scrolling="0"
```
Comment (nikolasrieble) [inline src/views/IndexView.js:79]: How is this change motivated? It seems unrelated.

File: src/locale/zh-CN.js (hunk 1 above)
Comment (ckifer) [reply src/locale/zh-CN.js:37]: Agreed sorry, this was laziness on my part

File: src/views/Storybook.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import './Storybook.scss';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const modules = ['installation', 'getting-started', 'customize'];
```
Comment (nikolasrieble) [inline src/views/Storybook.js:6]: Interesting, why define an array of 3, and then only use the first element?

File: src/views/Storybook.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+iframe.storybook {
+  width: 100%;
+  height: calc(100vh - 150px);
```
Comment (nikolasrieble) [inline src/views/Storybook.scss:3]: Any motivation for these specific values?

File: src/views/IndexView.js (hunk 2 above)
Comment (ckifer) [reply src/views/IndexView.js:79]: they are deprecated attributes https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#deprecated_attributes

File: src/views/Storybook.js (hunk 3 above)
Comment (ckifer) [reply src/views/Storybook.js:6]: tbh this was a copy paste from the other PR (and other components) I didn't look into implications

File: src/views/IndexView.js (hunk 2 above)
Comment (ckifer) [reply src/views/IndexView.js:79]: added some CSS to replace these deprecated attrs

**Review by nikolasrieble — APPROVED** [review]
Great, thank you!
